docs/user-guide/plugins.md

```diff
@@ -246,11 +244,21 @@ event.
     Parameters:
     : __page:__ `mkdocs.nav.Page` instance
     : __config:__ global configuration object
-    : __site_navigation__: global navigation object
+    : __site_navigation:__ global navigation object
 
     Returns:
     : `mkdocs.nav.Page` instance
 
+##### on_load_page_markdown
+
+:   The `load_page_markdown` event can replace the default loading mechanism to 
+    load the contents of the file contents. If the plugin(s) return `None`, the
+    default loading will be performed.
+
+    Parameters:
+    : __page:__ `mkdocs.nav.Page` instance
+    : __config:__ global configuration object
+
```

Comment:
Just noticed this is missing the `Returns`  description which exists in the other events. Perhaps the description should specify that the Markdown string should include any meta-data (if it exists) as that has not yet been removed.

Reply:
Perhaps:

    Returns:
        : Content of file as string of Markdown text with any meta-data still attached.

Reply:
thanks for spotting. fixed it.